### PR TITLE
[doc] YBA Installer - Identify settings that can't be reconfigured #19893

### PR DESCRIPTION
**Pull request by ddhodge**
#19893 highlight which configuration options can't be changed after install

- installRoot
- postgres.useExisting
- service_username
- postgres.install.ldap_enabled

@netlify /preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/#configuration-options

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 1]
```diff
@@ -519,19 +521,24 @@ PostgreSQL configuration is divided into two different subsections:
 
 These options are mutually exclusive, and can be turned on or off using the _enabled_ option. Exactly one of these two sections must have enabled = true, while the other must have enabled = false.
 
+Note that you can't change the `useExisting` setting after installation.
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:524]: it's postgres.useExisting.enabled and postgres.install.enabled. They can't change either of the settings mentioned above after initial install. Along with install.ldap_enabled.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 2]
```diff
@@ -460,53 +460,55 @@ You can set the following YBA Installer configuration options.
 | `server_cert_path`<br />`server_key_path` | If providing custom certificates, give the path with these values. If not provided, the installation process generates self-signed certificates. Optional. |
 | `service_username` | The Linux user that will run the YBA processes. Default is `yugabyte`. The install process will create the `yugabyte` user. If you wish to use a different user, create that user beforehand and specify it in `service_username`. YBA Installer only creates the `yugabyte` user, not custom usernames. |
 
+Note that you can't change the `installRoot` and `service_username` settings after installation.
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:463]: Can' we star them in the table or something and then the star points down here? It's kind of hidden and might not be as clear.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 3]
```diff
@@ -519,19 +521,24 @@ PostgreSQL configuration is divided into two different subsections:
...
+Note that you can't change the `useExisting` setting after installation.
+
 **Install options**
 
 | Option | Description |
 | :--- | :--- |
-| `port` | Port PostgreSQL is listening to.
-| `restartSecond` | Wait time to restart PostgreSQL if the service crashes.
-| `locale` | locale is used during initialization of the db.
+| `port` | Port PostgreSQL is listening to. |
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:530]: There's an "enabled" boolean field for both the install options and useExisting options. https://github.com/yugabyte/yugabyte-db/blob/d6df95227d59cca5950f66f027372a4fc244564b/managed/yba-installer/yba-ctl.yml.reference#L152 This is what they can't change post install, and we should do the same star or italicize in the table and then point them to the note that says those fields can't be changed.

**Review by chidmuthu — CHANGES_REQUESTED** [review]
Left individual comments

**Review by aishwarya24 — APPROVED** [review]
LGTM. Backport to stable and other applicable versions.

**Pull request merged**